documentation/manual/src/main/asciidoc/modules/mongodb.asciidoc

```diff
@@ -1563,6 +1563,47 @@ Wife collection as JSON in MongoDB
 ----
 ====
 
+.Bidirectional one-to-one (Mapping to the same Entity)
+====
+[source, JAVA]
+----
+@Entity
+public class Person {
+
+    @Id
+    private String id;
+    private String name;
```

Comment:
Let's keep the example simpler using the field `name` as `@Id` and removing the `field` `id`.

Reply:
On second thought adding a field role might make the example easier to understand.